src/app/pages/Privacy.svelte

```diff
@@ -0,0 +1,170 @@
+<script>
+  import Button from "../components/Button.svelte";
+</script>
+
+<style>
+  .privacy {
+    width: 50%;
+    margin: auto;
+    margin-top: 20px;
+  }
+
+  .privacy h1 h2 h3 h4 {
+    font-family: Cabin, sans-serif;
+  }
+
+  a {
+    text-decoration: none;
+  }
+</style>
+
+<main>
+  <div class="privacy">
+    <a href="/"><Button color="blue">Back</Button></a>
+    <h2>Predictive Text Studio Privacy Policy</h2>
+    <p><strong>Effective as of:</strong> 11/12/2020</p>
+    <h3>1. Introduction</h3>
+    <p>
+      These Privacy Policy (“Privacy Policy”) apply to the use of the website
+      and products provided by Predictive Text Stuio (hereinafter also referred
+      as “we” or “us”).
+    </p>
+    <p>
+      This Privacy Policy applies and has effect in respect of all services and
+      other products, software, made available by us, as well as any other
+      online features relating to the website and its content (the
+      “Service(s)”).
+    </p>
+    <p>
+      If you have any questions or comments about this Privacy Policy, please
+      contact us at
+      <a href="mailto:easantos@ualberta.ca">easantos@ualberta.ca</a>.
```

Comment:
[Using `+tag` helps me filter emails](https://gmail.googleblog.com/2008/03/2-hidden-ways-to-get-more-from-your.html)!

```suggestion
      <a href="mailto:easantos+ptstudio@ualberta.ca">easantos+ptstudio@ualberta.ca</a>.
```